### PR TITLE
correct methods name and update sample configuration file

### PR DESCRIPTION
**Pull request by sophiesongge**

### CODE REVIEW

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java [hunk 1]
```diff
@@ -774,7 +774,7 @@ public boolean restartInErrorTask(String jobId, String taskName)
             throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java [hunk 2]
```diff
@@ -759,7 +759,7 @@ public boolean finishInErrorTask(String jobId, String taskName)
...
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 76 issues
* ![MINOR][MINOR] 76 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [SchedulerFrontend.java#L452](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L452): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L484](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L484): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L494](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L494): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L505](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L505): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L540](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L540): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L550](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L550): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L560](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L560): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L567](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L567): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L589](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L589): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L599](https://github.com/ow2-proactive/scheduling/blob/f7a17b74548ccc3c5345c564f23beb0c33869a11/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L599): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java (hunk 1 above)
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 76 issues
* ![MINOR][MINOR] 76 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [SchedulerFrontend.java#L452](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L452): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L484](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L484): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L494](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L494): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L505](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L505): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L540](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L540): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L550](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L550): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L560](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L560): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L567](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L567): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L589](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L589): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L599](https://github.com/ow2-proactive/scheduling/blob/4609672dabbf6002844765e72204f8883c128857/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L599): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java (hunk 1 above)
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 76 issues
* ![MINOR][MINOR] 76 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [SchedulerFrontend.java#L452](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L452): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L484](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L484): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L494](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L494): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L505](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L505): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L540](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L540): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L550](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L550): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L560](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L560): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L567](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L567): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L589](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L589): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L599](https://github.com/ow2-proactive/scheduling/blob/f6cb3d2e43086e9e9db596c69dc96c6036d12404/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L599): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java (hunk 1 above)
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 76 issues
* ![MINOR][MINOR] 76 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [SchedulerFrontend.java#L452](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L452): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L484](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L484): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L494](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L494): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L505](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L505): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L540](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L540): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L550](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L550): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L560](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L560): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L567](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L567): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L589](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L589): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [SchedulerFrontend.java#L599](https://github.com/ow2-proactive/scheduling/blob/2c1ee0406227e721efc76e31e6d4a46b4a4856b5/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java#L599): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java (hunk 1 above)
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sophiesongge** [comment]
retest this please

**Pull request merged**